src/server.js

```diff
@@ -86,8 +47,7 @@ app.post("/search", (req, res) => {
     presentations,
   );
 
-  const html = render("search", { presentations: presentationsFound });
-  res.status(200).send(html);
+  res.render("search", { presentations: presentationsFound , layout: false});
```

Comment:
If `layout: false` is not provided, the html includes the layout, and the entire html document. This breaks the table:

<img width="948" alt="Screenshot 2024-06-28 at 6 40 23 AM" src="https://github.com/monicaleep/member-fe-microsite/assets/1879443/2f86e138-affa-4f6d-b3a0-cfa1cbf117db">

We'll need to follow this convention for all htmx requests, which expect html fragments, not html documents.